new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -148,8 +148,7 @@ public void beforeAll(ExtensionContext context) {
 
         // The following code is for pulling WLS images if running tests in Kind cluster
         if (KIND_REPO != null) {
-          // We can't figure out why the kind clusters can't pull images from OCR using the image pull secret. There
-          // is some evidence it may be a containerd bug. Therefore, we are going to "give up" and workaround the issue.
+          // The kind clusters can't pull images from OCR using the image pull secret. 
```

Comment:
```suggestion
          // The kind clusters can't pull images from OCR using the image pull secret. It may be a containerd bug, we are going to workaround the issue
```